<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>traversegraph</class>
 <widget class="QDialog" name="traversegraph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_home">
      <widget class="QLabel" name="label_graphHome">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>10</y>
         <width>351</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Traverse Graph&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_backGraph">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>420</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_graphImage">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>471</width>
         <height>221</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>350</y>
         <width>251</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>&lt;Select Here&gt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 - Minimum Spanning Tree</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 - Depth-First Search (from Hard Rock Stadium)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 - Breadth-First Search (from Lambeau Field)</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="page_MST">
      <widget class="QLabel" name="label_titleMST">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>531</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Minimum Spanning Tree&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_backMST">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>420</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_DFS">
      <widget class="QLabel" name="label_titleDFS">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>531</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Depth-First Search&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_backDFS">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>420</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget_DFS">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>100</y>
         <width>231</width>
         <height>251</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Stadium Name</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_distanceDFS">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>370</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_BFS">
      <widget class="QLabel" name="label_titleBFS">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>531</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Breadth-First Search&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_backBFS">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>420</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget_BFS">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>100</y>
         <width>231</width>
         <height>251</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Stadium Name</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_distanceBFS">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>370</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
